packages/strapi-plugin-content-manager/test/relation-list.test.e2e.js

```diff
@@ -0,0 +1,183 @@
+    data.shops.push(res.body);
+  }
+
+  for (const product of products) {
+    const res = await rq({
+      method: 'POST',
+      url: 'content-manager/explorer/application::product.product',
+      body: {
+        ...product,
+        shops: [data.shops[0].id],
+      },
+    });
+    data.products.push(res.body);
+  }
+
+  if (publishAProduct) {
+    const res = await rq({
+      method: 'POST',
+      url: `/content-manager/explorer/application::product.product/publish/${data.products[0].id}`,
+    });
+    data.products[0] = res.body;
+  }
+}
+
+async function deleteFixtures() {
+  for (let shop of data.shops) {
+    await rq({
+      method: 'DELETE',
+      url: `/content-manager/explorer/application::shop.shop/${shop.id}`,
+    });
+  }
+  for (let product of data.products) {
+    await rq({
+      method: 'DELETE',
+      url: `/content-manager/explorer/application::product.product/${product.id}`,
+    });
+  }
+}
+
+describe('Relation-list route', () => {
+  beforeAll(async () => {
+    const token = await registerAndLogin();
+    rq = createAuthRequest(token);
+
+    modelsUtils = createModelsUtils({ rq });
+  }, 60000);
+
+  describe('without draftAndPublish', () => {
+    beforeAll(async () => {
+      await modelsUtils.createContentTypes([productModel, shopModel]);
+      await createFixtures();
+    }, 60000);
+
+    afterAll(async () => {
+      await deleteFixtures();
+      await modelsUtils.deleteContentTypes(['product', 'shop']);
+    }, 60000);
+
+    test('Can get relation-list for products of a shop', async () => {
+      const res = await rq({
+        method: 'GET',
+        url: '/content-manager/explorer/application::shop.shop/relation-list/products',
+      });
+
+      expect(res.body).toHaveLength(data.products.length);
+      data.products.forEach((product, index) => {
+        expect(res.body[index]).toStrictEqual(_.pick(product, ['_id', 'id', 'name']));
+      });
+    });
+  });
+
+  describe('with draftAndPublish', () => {
+    beforeAll(async () => {
+      await modelsUtils.createContentTypes([productWithDPModel, shopModel]);
+      await createFixtures({ publishAProduct: true });
+    }, 60000);
+
+    afterAll(async () => {
+      await deleteFixtures();
+      await modelsUtils.deleteContentTypes(['product', 'shop']);
+    }, 60000);
+
+    test('Can get relation-list for products of a shop', async () => {
+      const res = await rq({
+        method: 'GET',
+        url: '/content-manager/explorer/application::shop.shop/relation-list/products',
+      });
+
+      expect(res.body).toHaveLength(data.products.length);
+
+      const tomatoProductRes = res.body.find(p => p.name === 'tomato');
+      const appleProductRes = res.body.find(p => p.name === 'apple');
+
+      expect(tomatoProductRes).toMatchObject(_.pick(data.products[0], ['_id', 'id', 'name']));
+      expect(tomatoProductRes.published_at).not.toBeNull();
+      expect(isNaN(new Date(tomatoProductRes.published_at).valueOf())).toBe(false);
```

Comment:
Maybe add a comment or even an utility somewhere to explain how you test if this is a valid date?